index.js

```diff
@@ -0,0 +1,132 @@
+// dependencies
+const inquirer = require("inquirer");
+const fs = require("fs");
+const createHTML = require("./src/html-creator.js");
+
+const Employee = require("./lib/employee.js");
+const Manager = require("./lib/manager.js");
+const Intern = require("./lib/intern.js");
+const Engineer = require("./lib/engineer.js");
+
+const staff = [];
+
+// function to get user input and create new employee object
+const getEmployeeInfo = (role) => {
+  return inquirer
+    .prompt([
+      {
+        type: "input",
+        message: `What is the ${role}'s name?`,
+        name: "name",
+      },
+
+      {
+        type: "number",
+        message: `What is the ${role}'s ID?`,
+        name: "id",
+      },
+
+      {
+        type: "input",
+        message: `What is the ${role}'s email?`,
+        name: "email",
+      },
+
+      {
+        type: "input",
+        message: "What is the manager's office location?",
+        name: "officeLocation",
+        when: role === "manager",
+      },
+
+      {
+        type: "input",
+        message: "What is the engineer's GitHub username?",
+        name: "github",
+        when: role === "engineer",
+      },
+
+      {
+        type: "input",
+        message: "What is the intern's school?",
+        name: "school",
+        when: role === "intern",
+      },
+
+      {
+        type: "list",
+        message: "What do you wish to do next?",
+        name: "nextStep",
+        choices: ["Add an engineer", "Add an intern", "Build my team"],
+      },
+    ])
+
+    .then((response) => {
```

Comment:
It would be nice to see you using the `async` `await` syntax now - I think i've noted this in previous homeworks of yours, but it makes the code much easier to follow and it should be easier for you to write too.